apps/web/src/app/global-error.tsx

```diff
@@ -0,0 +1,35 @@
+'use client'
```

Comment:
We already have a global error I think

Reply:
Nein we have an error for private pages

Reply:
And Sentry catch errors that are catched by Private pages ErrorBoundary?

Reply:
yes this is just to make sure sentry also catches any unexpected error outside that boundary